imap_data_access/file_validation.py

```diff
@@ -26,21 +26,19 @@ def __init__(self, filename: str | Path):
         path is set by the "IMAP_DATA_DIR" environment variable, or defaults to "data/"
 
         Current filename convention:
-        <mission>_<instrument>_<datalevel>_<descriptor>_<start_date>_<end_date>
+        <mission>_<instrument>_<datalevel>_<descriptor>_<start_date>-<repointing?>
         _<version>.<extension>
 
         NOTE: There are no optional parameters. All parameters are required.
         <mission>: imap
-        <instrument>: idex, swe, swapi, hi-45, ultra-45 and etc.
-        <datalevel> : l1a, l1b, l1, l3a and etc.
+        <instrument>: codice, glows, hi, hit, idex, lo, mag, swapi, swe, ultra
+        <data_level> : l1a, l1b, l1, l3a and etc.
         <descriptor>: descriptor stores information specific to instrument. This is
             decided by each instrument. For L0, "raw" is used.
-        <startdate>: startdate is the earliest date in the data. Format - YYYYMMDD
-        <enddate>: Some instrument and some data level requires to store date range.
-            If there is no end date, then startdate will be used as enddate as well.
-            Format - YYYYMMDD.
-        <version>: This stores software version and data version. Version format is
-            vxx-xx.
+        <start_date>: startdate is the earliest date in the data, format: YYYYMMDD
+        <repointing?>: This is an optional field. It is used to indicate which
+            repointing the data is from, format: XXXXX
```

Comment:
```suggestion
        <repointing>: This is an optional field. It is used to indicate which
            repointing the data is from. Repointing format is repoint followed by 5-digits: repointXXXXX
```

Reply:
We had a discussion on Slack about prefixing repoint to give it some meaning like how version number starts with `vxxx`. You mentioned about doing something like this for repointing, `repointXXXXX`. Do we still want to do this? That's why I suggested above change.

Reply:
I added the `repoint` qualifier, but left off the extra text in the format section.